cmd/gen/renovate/command.go

```diff
@@ -0,0 +1,56 @@
+package renovate
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+)
+
+const (
+	name        = "renovate"
+	description = "Generates Renovate config for go and docker dependencies (renovate.yml)."
```

Comment:
Direction is nice and we've tested the Go support a fair bit. My main concern is about the repos using Python for tests.

We use pip with dependabot like here

https://github.com/giantswarm/nginx-ingress-controller-app/blob/28548c426b944a63c97302ecf8148bd4c77ea862/.github/dependabot.yml#L1

Have you had a chance to test the Python support? Maybe Halo can helm with that.

Reply:
renovate also support pip dependency upgrade. https://docs.renovatebot.com/python/ 
But I haven't had any chance to test this in python. Actually, I'm thinking that using renovate only for the team who opt-in by checking https://github.com/giantswarm/github/blob/master/repositories/meta.yaml. We can first use renovate from go project and slowly roll out to the different projects who use a different language other than go.  WDYT?

Reply:
> Actually, I'm thinking that using renovate only for the team who opt-in by checking https://github.com/giantswarm/github/blob/master/repositories/meta.yaml. We can first use renovate from go project and slowly roll out to the different projects who use a different language other than go.

Yes, good idea. Better to make it opt-in and then teams can migrate at their own pace and try out support for other languages.